Tools/px_uploader.py

```diff
@@ -336,7 +336,11 @@ def __determineInterface(self):
         self.port.flushInput()
         # Set a baudrate that can not work on a real serial port
         # in that it is 233% off.
-        self.port.baudrate = self.baudrate_bootloader * 2.33
+        try:
+            self.port.baudrate = self.baudrate_bootloader * 2.33
+        except Exception as e:
```

Comment:
What exception do you get here? Can we use a more specific one?

Reply:
I thought about that but bad excuse: my knowledge in Python is too rusty to quickly figure it out quickly 😇. The string `str(e)` is `non-standard baudrates are not supported on this platform` but we should test for the exception type and not the converted string and I have to read up how to check that properly.

Reply:
`print(type(e))` ?

Reply:
👍 
```
print(e.__class__.__name__)
print(type(e))
```
```
NotImplementedError
<type 'exceptions.NotImplementedError'>
```
Let me adjust.